base/tools/src/main/native/p12tool/p12tool.c

```diff
@@ -0,0 +1,1221 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
May I suggest you take a look at pki/base/tools/src/main/native/p7tool/p7tool.c where it states:
/* This file is based upon the file originally obtained from:
 * 
 *     CVSROOT=:pserver:anonymous@cvs-mirror.mozilla.org:/cvsroot
 *     cvs export -r NSS_3_11_3_RTM -N
 *     mozilla/security/nss/cmd/p7content/p7content.c
 */
===
Basically, it's a good practice to let anyone having to look at this code in the future know that this code was taken from another source and modified on.
Actually, if you want to be even more through, a brief description of what you did might help such as

1. moreoids.c was added to support AES_KEY_WRAP_KWP oids
2. p12tools.c is modifed to call SECU_RegisterDynamicOids() provided in moreoids.c
3. anything else?
I wish I added a brief description in p7tool.c  what was done, because I don't remember ;-).